From 502bc0e059421d62d15de35db928b95fe46328fc Mon Sep 17 00:00:00 2001
From: Roger Siddons <dizygotheca@ntlworld.com>
Date: Fri, 9 Oct 2015 15:45:23 +0100
Subject: [PATCH] MediaMonitor: Unlock mutex before quitting

Prevent Qt 5 issuing "QMutex: Destroying locked mutex" warning when FE exits

diff --git a/mythtv/libs/libmyth/mythmediamonitor.cpp b/mythtv/libs/libmyth/mythmediamonitor.cpp
index 18eeae1..f0ef9d3 100644
--- a/mythtv/libs/libmyth/mythmediamonitor.cpp
+++ b/mythtv/libs/libmyth/mythmediamonitor.cpp
@@ -58,6 +58,7 @@ void MonitorThread::run(void)
         m_Monitor->CheckDevices();
         m_Monitor->m_wait.wait(&mtx, m_Interval);
     }
+    mtx.unlock();
     RunEpilog();
 }
 
-- 
2.1.4

